source/_UIAHandler.py

```diff
@@ -45,6 +45,7 @@
 ]
 
 badUIAWindowClassNames=[
+	"Microsoft.IME.CandidateWindow.View",

```

Comment:
Please add a comment here that explains why this UIA implementation is considered bad.

Reply:
The code here were written by 
 @michaelDCurran 
 at https://github.com/nvaccess/nvda/commit/dbbaacea10ac1ec8c07dbe943535d86284546861
I do not know much about its meaning. 
But removing this line then candidate report will not work.
May be there are some confusing UIA events should be suppressed?